examples/detailed_example.py

```diff
@@ -0,0 +1,96 @@
+import json
+import logging
+import os
+
+import aioredis
+
+import sanic
+from sanic import Sanic
+
+
+ENV_VARS = ["REDIS_HOST", "REDIS_PORT",
+            "REDIS_MINPOOL", "REDIS_MAXPOOL",
+            "REDIS_PASS", "APP_LOGFILE"]
+
+app = Sanic(name=__name__)
+
+logger = None
+
+
+@app.middleware("request")
+async def attach_db_connectors(request):
+    if not hasattr(request.app.config, "REDIS"):
+        logger.info("Setting up connection to Redis Cache")
+        request.app.config.REDIS = await aioredis.create_pool((app.config.REDIS_HOST, int(app.config.REDIS_PORT)),
```

Comment:
Ah... thanks for that. Was only meaning for that to be made once.
I can fix this.